configurations/test_pipeline_configuration.py

```diff
@@ -8,9 +8,12 @@
 PIPELINE_CONFIGURATION = PipelineConfiguration(
     pipeline_name="engagement-db-test",
     # TODO: store in messages and individuals_filter list of functions.
-    project_start_date=isoparse("2021-03-01T10:30:00+03:00"),
-    project_end_date=isoparse("2100-01-01T00:00:00+03:00"),
-    filter_test_messages=False,
+    project_start_date = isoparse("2021-03-01T10:30:00+03:00"),
+    project_end_date = isoparse("2100-01-01T00:00:00+03:00"),
+    filter_test_participants = False,
+    test_participant_uids = [
+        "avf-participant-uuid-51c15546-58a0-4ab1-b465-e65b71462a8f"
```

Comment:
```suggestion
    project_start_date=isoparse("2021-03-01T10:30:00+03:00"),
    project_end_date=isoparse("2100-01-01T00:00:00+03:00"),
    filter_test_participants=False,
    test_participant_uids=[
        "avf-participant-uuid-51c15546-58a0-4ab1-b465-e65b71462a8f"
```
for consistency with the rest of our this file, the rest of avf's code base, and the official python formatting standards

Reply:
Updated for consistency with the rest of this file - (If am not wrong pep8 recommends a single space on either side of the operator)